engine/apps/slack/scenarios/distribute_alerts.py

```diff
@@ -50,68 +46,85 @@
 logger.setLevel(logging.DEBUG)
 
 
-class AlertShootingStep(scenario_step.ScenarioStep):
+class IncomingAlertStep(scenario_step.ScenarioStep):
```

Comment:
curious on others thoughts around the naming change here? Personally `AlertShootingStep` (🔫) was not super clear to me on what it did